src/Backend/Modules/Analytics/Tests/Actions/IndexTest.php

```diff
@@ -9,7 +9,7 @@ class IndexTest extends BackendWebTestCase
 {
     public function testAuthenticationIsNeeded(Client $client): void
     {
-        self::assertAuthenticationIsNeeded($client, '/private/en/analytics/index');
+        $this->assertAuthenticationIsNeeded($client, '/private/en/analytics/index');
```

Comment:
👍  I like instance methods for phpunit, like Symfony describes in the docs https://symfony.com/doc/current/testing.html#unit-tests